index.bs

```diff
@@ -4266,17 +4266,24 @@ To <dfn>find matching sources</dfn> given an [=attribution trigger=] |trigger|:
     1. If |source|'s [=attribution source/reporting origin=] and |trigger|'s [=attribution trigger/reporting origin=] are not [=same origin=], [=iteration/continue=].
     1. If |source|'s [=attribution source/expiry time=] is less than or equal to |trigger|'s [=attribution trigger/trigger time=], [=iteration/continue=].
     1. [=list/Append=] |source| to |matchingSources|.
-1. Set |matchingSources| to the result of [=list/sort in descending order|sorting=] |matchingSources|
-    in descending order, with |a| being less than |b| if any of the following are true:
-      * |a|'s [=attribution source/priority=] is less than |b|'s [=attribution source/priority=].
+1. Let |sourceToAttribute| be null.
+1. An [=attribution source=] |a| <dfn for="attribution source">is higher-priority than</dfn> an [=attribution source=] |b|
```

Comment:
Let's move this out of this algorithm into its own definition section for clarity, and perhaps get rid of the `any of the following...` structure and instead make the order of the checks more clear, e.g.

```
1. If |a|'s [=attribution source/priority=] is greater than |b|'s [=attribution source/priority=], return true.
2. If |a|'s [=attribution source/priority=] is less than |b|'s [=attribution source/priority=], return false.
3. <!-- source time comparison -->
```